pastis/temporal_analysis/low_mode.ipynb

```diff
@@ -0,0 +1,656 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e93ffb1f",
+   "metadata": {},
+   "source": [
+    "## This notebook contains e2e analysis used to allocate tolerances for each $\\color{red}{\\text{Global Zernike Aberration}}$ mode for a segmented telescope."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d6c7adc8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.chdir(\"/Users/asahoo/repos/PASTIS\")\n",
+    "import time\n",
+    "from shutil import copy\n",
+    "from astropy.io import fits\n",
+    "import astropy.units as u\n",
+    "import hcipy\n",
+    "import numpy as np\n",
+    "import pastis.util as util    \n",
+    "from pastis.config import CONFIG_PASTIS \n",
+    "from pastis.e2e_simulators.luvoir_imaging import LuvoirA_APLC \n",
+    "from pastis.e2e_simulators.generic_segmented_telescopes import SegmentedAPLC\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "from scipy.interpolate import griddata\n",
+    "import exoscene.image\n",
+    "import exoscene.star\n",
+    "import exoscene.planet\n",
+    "from exoscene.planet import Planet\n",
+    "from astropy.io import fits as pf\n",
+    "from matplotlib.colors import TwoSlopeNorm\n",
+    "import matplotlib.gridspec as gridspec\n",
+    "from pastis.analytical_pastis.temporal_analysis import req_closedloop_calc_batch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3b633c0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coronagraph_design = 'small'\n",
+    "nb_seg = CONFIG_PASTIS.getint('LUVOIR', 'nb_subapertures')\n",
+    "nm_aber = CONFIG_PASTIS.getfloat('LUVOIR', 'calibration_aberration') * 1e-9\n",
+    "sampling = CONFIG_PASTIS.getfloat('LUVOIR', 'sampling')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "be9a64a1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_dir = \"/Users/asahoo/Documents/ultra/global_zernike\"\n",
+    "repo_dir = \"/Users/asahoo/repos/PASTIS\"\n",
```

Comment:
use the function from `utils` we have for this please. although in this case, you don't even use this variable so delete it.